microservice-consul-demo/microservice-consul-demo-catalog/pom.xml

```diff
@@ -13,6 +13,11 @@
 	<dependencies>
 
 		<dependency>
+			<groupId>org.springframework.boot</groupId>
```

Comment:
Maybe that is pedantic but I prefer adding the optional spring-boot devtools to the end of the dependencies list.